src/main/java/edu/hm/hafner/analysis/parser/DoxygenParser.java

```diff
@@ -14,7 +14,7 @@
     /**
      * Creates a new instance of {@link DoxygenParser}.
      */
-    public DoxygenParser() {
+    DoxygenParser() {
```

Comment:
This one needs to be public, because we call it in the warnings plugin!

Reply:
:+1: